modules/ossm-cr-general.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-general_{context}"]
+= general parameters
+
+Here is an example that illustrates the `spec.general` parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
+
+.Example general parameters
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec
+  general:
+      logging:
+        componentLevels: {}
+          # misc: error
+        logAsJSON: false
+      validationMessages: true
+----
+
+
+.Istio general parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|logging:
+|Use to configure logging for the control plane components.
+|
+|
+
+|logging:
+ componentLevels:
+|Use to specify the component logging level.
+|Possible values: `trace`, `debug`, `info`, `warning`, `error`, `fatal`, `panic`.
```

Comment:
we should maybe add a list of components here as well. they are:

ads, adsc, all, analysis, authn, authorization, ca, cache, cli, default, installer, klog, mcp, model, patch, processing, resource, source, spiffe, tpath, translator, util, validation, validationController, wle ([core istio.io](https://istio.io/latest/docs/ops/diagnostic-tools/component-logging/))

and

federation (maistra.io)

Reply:
that's right. to be honest, I had some issues while testing these log components (some of them work on istiod but break proxy). let's just leave them out